backend/models/user.model.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+let Workout = require('./workout.model');
+
+var userSchema = new Schema({
+    name:  {
+        type: String,
+        required: [true, "Please enter your name"],
```

Comment:
What does this do? Is it serving as the default value if one isn't provided? If so, keep in mind that we don't handle user inputs, we are assuming all of that has been handled prior to passing into the object I think